INSTALLATION.md

```diff
@@ -43,13 +43,22 @@ We have tried to make the process simple. Here's what you need to do.
 
 ## Developer Environment Setup
 
-There are multiple ways to access the Talawa-API that the Talawa mobile app uses.
+Talawa mobile app requires [talawa-api](https://github.com/PalisadoesFoundation/talawa-api) running locally or remotely, which will be used as an organisation url. 
 
-### On Your Local Machine
+#### On Your Local Machine
 
-You can also setup your own local instance of [Talawa-API](https://github.com/PalisadoesFoundation/talawa-api). The advantage is that you'll be working with the latest code.
+You need to setup your own local instance of [Talawa-API](https://github.com/PalisadoesFoundation/talawa-api) and [Talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin). The advantage is that you'll be working with the latest code.
 
-1. If you have set up all the 3 projects locally ([talawa](https://github.com/PalisadoesFoundation/talawa), [talawa-api](https://github.com/PalisadoesFoundation/talawa-api), and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin)), then use the URL (Organization URL) in this form:
-    1. http://(localhost IP):4000/graphql (See the next item for the IP addresses to use for your environment)
-1. Every Emulator has a different localhost IP address. 
-    1. For the Android Studio emulator use: http://10.0.2.2:4000/graphql
+1. You need to setup all the 3 projects locally ([talawa](https://github.com/PalisadoesFoundation/talawa/blob/develop/INSTALLATION.md), [talawa-api](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md), and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/INSTALLATION.md)).
+2. After setting up [talawa-api](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md) and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/INSTALLATION.md) create an account in talawa admin.
```

Comment:
Please find a way to exclude links that include a specific branch. If the default branch changes, then the documentation will be incorrect.